src/utils/fund_utils.py

```diff
@@ -1,8 +1,46 @@
 import re
+from typing import Dict
+
+from .colors import RgbColor, HlsColor
+
+FUND_GROUP_BASE_COLORS: Dict[str, RgbColor] = {
+    "PASSIVE": RgbColor(red=200, green=40, blue=40),
+    'ACTIVE': RgbColor(red=200, green=40, blue=40),
+    'BALANCED': RgbColor(red=230, green=130, blue=0),
+    'FIXED_INCOME': RgbColor(red=80, green=140, blue=80),
+    'OTHER': RgbColor(red=100, green=100, blue=100)}
 
 
 class FundUtils:
 
+    @staticmethod
+    def get_fund_color(fund_group: str, risk_level: int) -> RgbColor:
+        """
+        Creates fund based on fund group and risk level
+        Args:
+            fund_group: fund group
+            risk_level: risk level
+
+        Returns:
+            Fund color
+        """
+        base_color = FundUtils.get_fund_group_base_color(fund_group=fund_group)
+        color = base_color.to_hls()
+        lightness_add = ((7 - min(max(1, risk_level), 7)) * 0.07)
```

Comment:
so if risk is 7 or more , lightness_add is 0. so I assume that base color for these funds are different? Because if they are the same, then fund color for them will be the same.

Reply:
risk should always be between 1 - 7, this is just a failsafe to ensure that the value will be always within that range